packages/kit/src/views/Discovery/pages/SearchModal/index.tsx

```
@@ -101,18 +102,49 @@ function SearchModal() {
         setSearchList([]);
         return;
       }
+      
```

ESLint

[error] Delete `······`

(prettier/prettier)